package/yast2-ruby-bindings.spec

```diff
@@ -17,7 +17,7 @@
 
 
 Name:           yast2-ruby-bindings
-Version:        3.1.52
+Version:        3.1.51.2
```

Comment:
MMM what will happen if we do a fix for SLE-12-SP2? and we unified to 3.1.52 instead of 3.1.53?